packages/cerebral-website-tutorial/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "cerebral-tutorial",
+  "version": "0.1.0",
+  "private": true,
+  "author": "Philipp Knecht <fopsdev@gmail.com>",
+  "contributors": [
+    "Aleksey Guryanov <gurianov@gmail.com>",
+    "Christian Alfoni <christianalfoni@gmail.com>"
+  ],
+  "devDependencies": {
+    "react-scripts": "0.7.0",
+    "shelljs": "^0.7.5"
+  },
+  "dependencies": {
+    "cerebral": "^2.0.0-0",
+    "cerebral-provider-http": "^1.0.0-0",
+    "cerebral-router": "^1.0.0-0",
+    "js-logger": "1.3.0",
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2"
+    
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "ch01": "cd ./parts/01 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
```

Comment:
`rm -rf` and `cp -R` will not work on windows. let add `rimraf` as dev dep here. also smth cp should exist as well